custom_components/kumo/climate.py

```diff
@@ -528,3 +529,14 @@ def set_fan_mode(self, fan_mode):
 
         response = self._pykumo.set_fan_speed(fan_mode)
         _LOGGER.debug("Kumo %s set fan speed response: %s", self._name, response)
+
+    def turn_on(self) -> None:
+        """Turn the climate on. This implements https://www.home-assistant.io/integrations/climate/#action-climateturn_on.
+
+        For now, turn it on to HVACMode.COOL which always exists in self._hvac_modes.
```

Comment:
Sorry, this is unintuitive and incorrect behavior. I'd rather not implement "turn_on" at all.

Reply:
That's fair